runhouse/resources/hardware/on_demand_cluster.py

```diff
@@ -446,19 +446,6 @@ def up(self):
 
         self.restart_server()
 
-        # Save a limited version of the local ~/.rh config to the cluster with the user's hashed token
```

Comment:
Why are you not sending this anymore?

Reply:
Moved it into the restart_server() logic